### ISSUE TITLE
cannot exit from kubectl exec (openshift 3.11)

### ISSUE THREAD

**Issue by jsirianni** [containers/container-selinux#94]
Hi, when I use the following selinux configuration:
type: `container_logreader_t`

I cannot exit from a container shell. Example:
1) oc deploy -f dpeloyment.yaml
2) oc exec -it <pod> sh
3) exit  

The exit hangs. The interesting part, I have read access to /var/log, which is what I require. 

In addition, I cannot delete the containers. They hang in a terminating state. I think it is the same issue.

audit.log gives this message when I try to exit from the container's shell.
```
type=AVC msg=audit(1586919161.042:18459): avc:  denied  { sigchld } for  pid=15994 comm="docker-containe" scontext=system_u:system_r:container_logreader_t:s0:c4,c17 tcontext=system_u:system_r:container_runtime_t:s0 tclass=process permissive=0
```

This is solved by disabling selinux, everything works great, however, my environment is under heavy compliance.

This may not be the correct place to file an issue. I am looking for guidance on how to bring this issue to the attention of the OpenShift maintainers. Does the issue give anyone any clues?

**Comment by jsirianni** [comment]
`spc_t` type, revolves the issue but I fear it is too permissive as well. At least it is only for the deployment in question.

**Comment by rhatdan** [comment]
Weird I have never seen this before.  I would have figured this would break lots of containers.

Fixed in container-selinux-2.132.0
